systems/controllers/osc/operational_space_control.cc

```diff
@@ -405,20 +413,37 @@ VectorXd OperationalSpaceControl::SolveQp(
   ///     mu_*lambda_c(3*i+2) + lambda_c(3*i+1) >= 0
   ///                           lambda_c(3*i+2) >= 0
   if (body_index_.size() > 0) {
+    VectorXd mu_minus1(2);
+    VectorXd mu_plus1(2);
     VectorXd inf_vectorxd(1); inf_vectorxd << numeric_limits<double>::infinity();
     for (unsigned int i = 0; i < active_contact_flags.size(); i++) {
       // If the contact is inactive, we assign zeros to A matrix. (lb<=Ax<=ub)
       if (active_contact_flags[i]) {
-        friction_constraints_.at(5 * i)->UpdateLowerBound(VectorXd::Zero(1));
-        friction_constraints_.at(5 * i + 1)->UpdateLowerBound(VectorXd::Zero(1));
-        friction_constraints_.at(5 * i + 2)->UpdateLowerBound(VectorXd::Zero(1));
-        friction_constraints_.at(5 * i + 3)->UpdateLowerBound(VectorXd::Zero(1));
+        if(time_since_last_state_switch < period_of_low_friction_[i]){
+          mu_minus1 << mu_low_friction_[i], -1;
+          mu_plus1 << mu_low_friction_[i], 1;
+        } else {
+          mu_minus1 << mu_, -1;
+          mu_plus1 << mu_, 1;
+        }
+        friction_constraints_.at(5 * i)->UpdateCoefficients(
```

Comment:
Maybe add a variable with value 5 to make it clearer where the 5 comes from?

Reply:
It came from the five constraints for each contact point: 
https://github.com/DAIRLab/dairlib/blob/82df28f7f513c45ec84a41f8a0071f9b4a6a8c7c/systems/controllers/osc/operational_space_control.cc#L405-L409
I added a piece of comment in the code explaining where it comes from.